test/reference-harness.js

```diff
@@ -28,7 +28,7 @@ export default {
       title: 'Test Post',
       description: 'Place we can call to test POST',
       scopes: [],
-      input: 'http://schemas.taskcluster.net/nothing.json'
+      input: 'https://taskcluster.net/schemas/nothing.json'
```

Comment:
I don't think it matters for this library, but this would now be a relative uri, such as `v1/nothing.json`.